### DISCUSSION TITLE
On drag release overscroll deceleration

### DISCUSSION THREAD

**Discussion by moishinetzer** [Norserium/react-indiana-drag-scroll#68] [Ideas]
Is there any way we could get a feature that after releasing the scroll decelerates to a stop as opposed to stopping imediately? 
Similar to how mobile scrolling works.

**Comment by Norserium** [comment]
@moishinetzer, alas, it's not available by default. [The associated issue](https://github.com/Norserium/react-indiana-drag-scroll/issues/5).

**Reply by moishinetzer** [reply]
I see... is there any interest in implementing it now? It's been quite some time since that issue's been active 😅

**Reply by Norserium** [reply]
@moishinetzer, well, I can't promise anything, because I'm currently spend pretty much free time on [`advanced-cropper`](https://github.com/advanced-cropper) project. But I will try to implement something in the new major release in this year. I see how many likes that issue have since it was created.

**Reply by moishinetzer** [reply]
Sounds exciting! Good luck with your other project

**Comment by Norserium** [comment]
This feature is implemented in [3.x.x](https://norserium.github.io/react-indiana-drag-scroll/) now.
